website/pages/docs/secrets/databases/index.mdx

```diff
@@ -69,32 +69,34 @@ management tool.
        password="..."
    ```
 
-   ~> It is highly recommended a user is created for Vault to use to
-      manage users rather than using the database's root account.
+   ~> It is highly recommended a user within the database is created
+      specifically for Vault to use. This user will be used to manipulate
+      dynamic and static users within the database. This user is called the
+      "root" user within the documentation.
 
    Vault will use the user specified here to create/update/revoke database
    credentials. That user must have the appropriate permissions to perform
-   actions upon other database users.
+   actions upon other database users (create, update credentials, delete, etc.).
 
    This secrets engine can configure multiple database connections. For details
    on the specific configuration options, please see the database-specific
    documentation.
 
-1. After configuring the user, it is highly recommended to rotate that user's
+1. After configuring the root user, it is highly recommended you rotate that user's
    password such that the vault user is not accessible by any users other than
    Vault itself:
 
    ```shell
    $ vault write -force database/rotate-root/my-database
    ```
 
-   When this is done, the password for the user specified in the previous step
-   is no longer accessible. Because of this, it is highly recommended that a
-   user is created specifically for Vault to use to manage database
-   users.
+   !> When this is done, the password for the user specified in the previous step
```

Comment:
Should this be `~>` or am I missing something? I'm fine if I'm missing something.

Reply:
`!>` is an error (red background) whereas `~>` is a warning (yellow background). Given how important this information is, I decided to make it red to grab people's attention more.

Reply:
Sounds good, I couldn't see that effect on the non-code view, so I wasn't sure. Thanks for clarifying!